### PR TITLE
Completed: HTML & CSS : Embedding Images and Video

### PR DESCRIPTION
**Pull request by Ferm0494**
index.html contains the youtube clone page.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,302 @@
...
+                <img class="youtubelogo" src="./assets/images/youtubelogo.png" alt="youtubelogo">
+
+                <div id="searchbar">
+                    <input id="input" type="text" placeholder="Search...">
+                    <button class="button"><i class="fa fa-search" aria-hidden="true">
+            </i></button>
+                </div>
+
+                <i class="fa fa-upload" aria-hidden="true"></i>
+                <button id="signin" class="button">Sign in</button>
```
Comment (adaorachi) [inline index.html]: - [ ] Here, you have all elements in the navbar hanging alone or in their different containers and which **should not** be. Your navbar should have three containers. The first one should contain the toggle-bar and logo and aligned to the leftmost part of the navbar. The second should contain the search bar and button and aligned to the center. And the third should have the icons aligned to the rightmost part. Kindly fix it.

**Image:**

theimage3

---

- [ ] There should be at least three embedded images on the rightmost part of the navbar that represent icons and/or an avatar [screenshot of requirement](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/embedding_images_and_video/icons_requirement.png)
---

- [ ] Make sure to use semantic HTML elements like the `<nav>` to enclose these elements.

Comment (adaorachi) [inline index.html]: You have an overuse of the `id` attribute in your codes. The purpose for which you used them can be interchanged with the `class` attribute and in fact, is preferred. The attribute `id` should be used only when it is needed and if the need is specific. If you need an element to have a unique identity, you can pass in multiple classes to that element with one of them having a unique name and use that class name for styling. Remove the `id` attributes and use `class `instead for those elements as the `id`'s are not necessary.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,302 @@
...
+
+        </header>
+        <main>
+
+            <!-- Youtube Main video -->
+            <div id="firstcolumn">
+                <iframe src="https://www.youtube.com/embed/lsuaFowZMjM" allowfullscreen></iframe>
+
+                <div id="principaldiv">
+                    <div>
```
Comment (adaorachi) [inline index.html]: - [ ] An overuse of the `id` attribute. Refer to my comment above and fix them. Also, search for other reoccurring usages of the `id` attributes and fix them.

---

- [ ] You also have the wrong use of your semantic HTML elements. The `<aside>` tag should not be nested in your`<main>` tag. The `<main>` section specifies/contains the main content of a document while the `<aside> `section defines the content aside from the page content (like a sidebar).

**Comment by adaorachi** [comment]
👋 Hi @Ferm0494 and @brugobi,
Good Job completing your first MIcroverse Project 👏 👏. You both did great in this project but there are a couple of changes you will need to implement to make your project look better and very professional.

**Kindly read through the following mandated requirements that you missed out on this project and implement the changes.**

---
### **CHANGES REQUIRED**
---

- [ ] Your index HTML code contains an error. Make sure to validate your codes and also at the end of these changes to fix the current error or any error that may occur afterward. Make use of this validating site - [validator.w3](https://validator.w3.org/#validate_by_upload).

---

- [ ] You need to fine-tune your README file to look more professional and appealing. Read through this instruction from this [pathwright link](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/)  to fix that.
---

- [ ] The view and appearance of your project should match that of this [youtube video player page](https://www.youtube.com/watch?v=V74l_zS1x8E). Remove the borders around the parent container and make it span the entire width of the page just like the original site. Compare the screenshots below or visit the link attached to this comment and make all necessary changes.

**Your Version:**

theimage1

**Original Version:**

theimage11

---

- [ ] The contents and recommended buttons under the video should be properly placed and aligned accordingly. Compare the screenshots below or reference the original site to implement this change.

**Your Version:** 
theimage2

**Original Version:**

theimage22

---

**Review by adaorachi — CHANGES_REQUESTED** [review]

**Comment by adaorachi** [comment]
### **Implement the changes requested above and request for another review by submitting a code review request.**

Should you have any questions or need any help, please feel free to reach out to me on Slack(@adaorachi). ☺️ 

**Happy Coding and Good-luck on it**

**MaryAnn Chukwuka**

**Comment by thsvr** [comment]
### **STATUS: PROJECT APPROVED** :1st_place_medal: :1st_place_medal: :1st_place_medal: 

Hi @brugobi and @Ferm0494 :smile_cat: :smile_cat: 

**Congratulations on the hard work, your project was approved** :fireworks: :fireworks:  :fireworks: 

 :rotating_light: :rotating_light:   **BEFORE MERGE THIS PROJEC INTO MASTER** :rotating_light: :rotating_light: 

:v:  Can you please update the live version and add a screenshot on the readme? The readme is like a business card of your project :smile_cat: 

### **Suggestion for the next projects:**

:v: As mentioned by the MaryAnn (TSE), an overuse of ID and it is not good practice, you can use classes as much as possible. So please consider it on the next project :smile_cat: 

:v:  On the pull request, it is required to add a short summary that describes what you guys did on the project or what you guys fixed after the review and before the submissions for another review. Can you please consider it on the next pull requests? 

:v: You can create a feature branch with a more descriptive name **on the next projects**, like ''main-page'' or ''youtube-page'' instead of ''feature'' or ''development'', for example. It will be more descriptive. The development branch has a special usage in GitFlow, you guys will use it on the next section but if you feel curious you can read more about it here -> [GitFlow](https://www.atlassian.com/git/tutorials/comparing-workflows/gitflow-workflow)


If you have any questions, don't hesitate to contact me through slack [@thaís Vieira Azevedo](url)

### **Happy coding and good luck on the next project** :computer: :computer: 

_Project reviewed by Thaís Vieira (TSE)_

**Pull request merged**